lib/emitter_wrapper.js

```diff
@@ -8,13 +8,8 @@ class EmitterWrapper {
 
   addListener (event, listener) {
     this.emitter.addListener(event, listener)
-
-    if (!this.listeners.hasOwnProperty(event)) {
-      this.listeners[event] = []
-    }
-
+    this.listeners[event] = this.listeners[event] || []
```

Comment:
Did you consider moving from object-as-dictionary to ES6 Map?

Reply:
I've tried, but had some issues. Furthermore I think we will be able to remove this class, because it's only used in one case:
https://github.com/karma-runner/karma/blob/4651524518df22842f69786f3f74fd4aeaf50c85/lib/server.js#L38

Probably we can use `process` directly in that cases - I'm not sure yet.